system-tests/README.md

```diff
@@ -178,7 +178,17 @@ Once completed, following services will start within their docker containers:
 
 (EDC Connectors will also be seeded with initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json))
 
-_Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts and `cli-tools` container will also automatically stop after registering participants._
+> Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts.
+
+> The container `cli-tools` will turn into the state `healthy` after registering successfully all participants.
+
+Sample for confirming successful run of container `cli-tools`.
+
+```powershell
+PS C:\> docker ps
+CONTAINER ID   IMAGE                                     COMMAND                   CREATED              STATUS                        PORTS                                                                              NAMES
```

Comment:
consider moving command outputs to separate ``` blocks, to make it clearer and avoid "random" syntax highlighting. E.g. Typora gives strange colours here

<img width="849" alt="image" src="https://user-images.githubusercontent.com/22099722/186336099-0eedb82d-fde9-4d10-9338-30ee5e2c6b88.png">

Reply:
this is a great approach and helps the commands to be clear and stand out in the `docs`

Reply:
Agreed! Thanks for sharing!